packages/config/config/devices/0x031e/lzw36.json

```
@@ -494,31 +540,77 @@
 			]
 		},
 		"25[0xff0000]": {
-			"label": "Fan LED Strip Effect Duration",
+			"label": "Fan LED Effect Duration",
 			"description": "Fan LED Strip Duration. 1 to 60 = seconds, 61 to 120 minutes, 121 - 254 = hours, 255 = Indefinitely Default: 0",
```

```suggestion
			"description": "Fan LED Strip Duration. 1 to 60 = seconds, 61 to 120 minutes, 121 - 254 = hours, 255 = Indefinitely",
```
There is some discrepancy here. The old description said default is 0, you define 255 below. And if 255 is correct, 0 has no meaning.

---

Please also check the other parameters that are similar.